renovate.json

```diff
@@ -102,6 +103,11 @@
       "extends": ["schedule:weekly"],
       "automerge": true,
       "separateMinorPatch": false
+    },
+    {
+      "descriptions": "Handle tag prefix in kustomize monorepo",
+      "matchPackageNames": ["kubernetes-sigs/kustomize"],
```

Comment:
please open a PR to the `.github` repo to add  this rule to the preset there.

Reply:
Sure! https://github.com/containerbase/.github/pull/9

Should I remove it here then?

Reply:
yes, remove it here

Reply:
done 😊